### ISSUE TITLE
Bug in Backup plugin - counting of free space vs. decision of enough space

### ISSUE THREAD

**Issue by JanSykor** [foosel/OctoPrint#2954]
#### What were you doing?

```
Creating the backup failed
OctoPrint could not create your backup. Please consult octoprint.log for details. Error:
```
In octoprint.log is message:

```
2018-12-11 16:38:14,556 - octoprint.plugins.backup - ERROR - Error while creating backup zip
Traceback (most recent call last):
File "/home/pi/oprint/lib/python2.7/site-packages/octoprint/plugins/backup/init.py", line 610, in _create_backup
raise InsufficientSpace()
InsufficientSpace
```
It happened if I have small space for backup of all - no matter if I wan´t save only setting excluded timelapses and gcodes.
If complete backup is smaller then free space, then it was created.

#### Version of OctoPrint
version 1.3.10 on Octopi 0.13.0 and RPi 3

#### Printer model & used firmware incl. version
Dual Builder printer  
Marlin V1; Sprinter/grbl mashup for gen6 FIRMWARE_URL:http://www.3dprinter4U.com PROTOCOL_VERSION:V2.2011H MACHINE_TYPE:Builder  EXTRUDER_COUNT:2

#### Browser and version of browser, operating system running browser
Google Chrome Version 71.0.3578.80 (Official Build) (32-bit)
Windows 10 Pro  v. 1709,  16299.849

#### Screenshot(s)/video(s) showing the problem:

I have read the FAQ.

**Comment by foosel** [comment]
This bug report is lacking a ton of information 🙄 Next time at the very least [post the link to the related forum post](https://discourse.octoprint.org/t/backup-plugin-setting/5795) so I have a chance to know what the problem even is - this here gives next to no context 😕

And this is already fixed in bf5bc14c8722a4e712cec0dfa2531a5005ed8650 (`maintenance`, soon `devel`, to be released in 1.3.11)

**Comment by foosel** [comment]
1.3.11 was released yesterday.
